core/templates/dev/head/editor/feedback_tab/FeedbackTab.js

```diff
@@ -89,6 +89,47 @@ oppia.controller('FeedbackTab', [
     });
   };
 
+  $scope.showSuggestionModal = function() {
+    $modal.open({
+      templateUrl: 'modals/editorViewSuggestion',
+      backdrop: true,
+      scope: $scope,
```

Comment:
Oh, I see! Can you use the 'resolve' pattern instead, and pass that into the controller? This is how we do it in other modals. In fact I think you can just pass in stateContent into the inner modal controller.

Also, I don't think you need to do any passing of $scope.activeThread outside the modal controller? As in, if you refer to $scope.activeThread outside the object passed to modal.open, the reference should be picked up.